spec/compiler/macro/macro_methods_spec.cr

```diff
@@ -20,6 +20,20 @@ describe "macro methods" do
       end
     end
 
+    describe "symbolize" do
+      it "expands macro with symbolize call on string" do
+        assert_macro "x", "{{x.symbolize}}", [StringLiteral.new("hello")] of ASTNode, ":\"\\\"hello\\\"\""
+      end
+
+      it "expands macro with symbolize call on symbol" do
+        assert_macro "x", "{{x.symbolize}}", [SymbolLiteral.new("hello")] of ASTNode, ":hello"
```

Comment:
This should probably give a symbol ":hello", that is, a symbol that has the colon. Right now if you have a string literal and call stringify on it, it will create a string literal whose contents are a string literal. The same should happen with symbolize.